lib/project-data-updater.js

```diff
@@ -44,9 +44,7 @@ function ProjectDataUpdater(config, repository, updateLock, done) {
 
   var that = this;
   this.done = done || function(err) {
-    if (err) {
-      console.log(err);
-    } else {
+    if (err === null) {
```

Comment:
I thought based on our discussion yesterday that error could just become a yes/no, as nothing else was calling done() that may be passing errors to it. If we both need to know success vs. not, AND error message, but those two aren't a one-to-one mapping, then I'll need to do more rearchitecting (but I'll also need to know who else is calling done() as it will require changing the method signature).